deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeGovernance.vue

```diff
@@ -37,7 +37,7 @@
           :src="`/${parentLocation}/static/images/discord.ico`"
           alt=""
           class="me-2 img-24px">
-        <span class="text-none">{{ $t('discussOnDiscord') }}</span>
+        <span class="text-none subtitle-1">{{ $t('discussOnDiscord') }}</span>
```

Comment:
You may need to change the `.v-btn` class in generic CSS of dapp to apply it globally (without `!important`).

Reply:
done!